<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>649</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>781</width>
    <height>649</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>781</width>
    <height>649</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>网络调试助手(V0.1)</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-color: rgb(170, 170, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>781</width>
      <height>601</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>781</width>
      <height>601</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>781</width>
      <height>601</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(192, 192, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>571</width>
       <height>401</height>
      </rect>
     </property>
     <widget class="QWidget" name="widget_5" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>571</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(248, 255, 188);</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>0</y>
         <width>267</width>
         <height>27</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>数据接收区</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_8">
          <property name="minimumSize">
           <size>
            <width>71</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>71</width>
            <height>21</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="receiveHex_checkBox">
          <property name="text">
           <string>16进制显示</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QTextBrowser" name="receive_textBrowser">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>571</width>
        <height>371</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>420</y>
       <width>571</width>
       <height>171</height>
      </rect>
     </property>
     <widget class="QWidget" name="widget_6" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>571</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(248, 255, 188);</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>1</x>
         <y>0</y>
         <width>460</width>
         <height>27</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="autoSend_checkBox">
          <property name="maximumSize">
           <size>
            <width>121</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>自动发送(ms)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="autoSendTime_lineEdit">
          <property name="minimumSize">
           <size>
            <width>61</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>61</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>1000</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>数据接收区</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_9">
          <property name="minimumSize">
           <size>
            <width>71</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>71</width>
            <height>21</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="sendHex_checkBox">
          <property name="text">
           <string>16进制显示</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPlainTextEdit" name="send_plainTextEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>1131</width>
        <height>211</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>183</width>
       <height>579</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>181</width>
          <height>401</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>181</width>
          <height>401</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 169, 255);
border-color: rgb(58, 68, 255);</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>180</width>
           <height>391</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="minimumSize">
             <size>
              <width>161</width>
              <height>131</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>161</width>
              <height>131</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="title">
             <string>通讯模式</string>
            </property>
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>40</y>
               <width>106</width>
               <height>98</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QRadioButton" name="tcpclient_radioButton">
                <property name="text">
                 <string>TCP Client</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="tcpserver_radioButton">
                <property name="text">
                 <string>TCP Server</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="udp_radioButton">
                <property name="text">
                 <string>UDP</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_10">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="minimumSize">
             <size>
              <width>161</width>
              <height>151</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>161</width>
              <height>151</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="title">
             <string>设置</string>
            </property>
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>30</y>
               <width>148</width>
               <height>125</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>远程主机</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="remoteIP_lineEdit">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>远程端口</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="remoteport_spinBox">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(255, 255, 255);</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>本地端口</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="localport_spinBox">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(255, 255, 255);</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>18</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="state_label">
              <property name="text">
               <string>通</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="connect_pushButton">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(234, 235, 214);</string>
              </property>
              <property name="text">
               <string>连接网络</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearReceive_pushButton">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(234, 235, 214);</string>
            </property>
            <property name="text">
             <string>清空接收区</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>181</width>
          <height>161</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 169, 255);
border-color: rgb(58, 68, 255);</string>
        </property>
        <widget class="Line" name="line_5">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>100</y>
           <width>118</width>
           <height>3</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>181</width>
           <height>161</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QCheckBox" name="rtSend_checkBox">
              <property name="text">
               <string>实时
发送</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="handSend_pushButton">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(234, 235, 214);</string>
              </property>
              <property name="text">
               <string>手动发送</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearSend_pushButton">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(234, 235, 214);</string>
            </property>
            <property name="text">
             <string>清空发送区</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="quit_pushButton">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(234, 235, 214);</string>
            </property>
            <property name="text">
             <string>退出</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuYywr">
    <property name="title">
     <string>文件(F)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>工具(T)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编码方式</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>关于(A)</string>
    </property>
   </widget>
   <addaction name="menuYywr"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
